haas/api.py

```diff
@@ -264,7 +264,7 @@ def node_power_off(node):
 
 
 @rest_call('PUT', '/node/<node>/bootdev', Schema({
-    'node': basestring, 'bootdev': Or('pxe','disk','none','bios'),
+    'node': basestring, 'bootdev': Or('pxe', 'disk', 'none'),
```

Comment:
If we're going to have this list be obm-specific, it needs to be specified elsewhere. this probably means going back to `basestring` and doing the validation inside the call to `node.obm.set_bootdev`.